share/zfsnap/core.sh

```diff
@@ -323,6 +402,50 @@ RmZfsSnapshot() {
     fi
 }
 
+SendSnapshots(){
+	#location of ssh key to used for authentication
+	local ssh_key=$1
+	#user to authenticate as
+	local user=$2
+	#ip or dns of remote server
+	local remote_server=$3
+	#remote filesystem in format of "zfs_pool_name/zfs_filesystem_name"
+	local remote_fs=$4
+	#local filesystem in format of "zfs_pool_name/zfs_filesystem_name"
+	local local_fs=$5
+	#common snapshot"
+	local common_snap=$6
+	#latest snapshot"
+	local latest_snap=$7
+	
+	#this sends the incrementals of all snapshots created since the last snapshot send to the backup server
+	#ouput of the zfs send and zfs receive commands is saved in temporary txt files to be sent to user
+	/bin/ssh -i $ssh_key $user@$remote_server /sbin/zfs send -vI $remote_fs@$common_snap $remote_fs@$latest_snap 2> /tmp/zfs_send_tmp.txt\
+	| /sbin/zfs receive -vF $local_fs &> /tmp/zfs_receive_tmp.txt
+		
+	#save the exit status of the two sides of pipe in variables and add them for a exit status total
+	local pipe1=${PIPESTATUS[0]} pipe2=${PIPESTATUS[1]} 
```

Comment:
Boo, is there another way I can capture errors here? I used to dump the error messages into txt files on /tmp, could do that, error and cleanup.